pycheribuild/config/compilation_targets.py

```diff
@@ -37,6 +37,7 @@
 from .loader import ConfigOptionBase
 from .target_info import (AutoVarInit, BasicCompilationTargets, CPUArchitecture, CrossCompileTarget, MipsFloatAbi,
                           TargetInfo)
+
```

Comment:
```suggestion
```

Reply:
still there

Reply:
This *still* hasn't been deleted

Reply:
I appear to have been very good at missing this :(